src/common/ConfigReader.cpp

```diff
@@ -133,21 +133,57 @@ namespace SDDM {
         return ret;
     }
 
-    void ConfigBase::load() {
-        // first check if there's at least anything to read, otherwise stick to default values
-        if (!QFile::exists(m_path))
-            return;
+    void ConfigBase::load()
+    {
+        //order of priority from least influence to most influence, is
+        //m_sysConfigDir in alphabetical order then m_configDir then m_path
```

Comment:
Something like Order of priority from least influence to most influence is system config dir, user config dir, legacy file